lib/hanami/cli/generators/app/slice/app_layout.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title><%= humanized_app_name %> - <%= humanized_slice_name %></title>
+    <%- if bundled_assets? -%>
+    <%%= favicon %>
```

Comment:
We'll need to add this back as an alias to Hanami::Helpers::AssetsHelper, which right now only has `#favicon_link_tag`. I can do that in a PR after this.

Reply:
@timriley Whoa, you were faster than my request. 😂 Thanks.